string object_name
---
#result definition
# Comes from moveit_msgs/MoveItErrorCodes
int32 error_code
geometry_msgs/PoseStamped object_pose
---
#feedback
